app/src/components/generic/recordCard/RecordCard.tsx

```diff
@@ -26,27 +23,21 @@ function RecordCard({
 }: Props) {
     const role = useRole();
     const userIsGpClinical = role === REPOSITORY_ROLE.GP_CLINICAL;
-    const [isLoading, setIsLoading] = useState(true);
     const mounted = useRef(false);
 
     useEffect(() => {
         const onPageLoad = async () => {
             await refreshRecord();
-            setIsLoading(false);
         };
         if (!mounted.current) {
+            onPageLoad();
             mounted.current = true;
-            void onPageLoad();
         }
     }, [refreshRecord]);
 
     const Record = () => {
-        if (isLoading) {
-            return (
-                <div className="pl-7">
-                    <ProgressBar status="Loading..." />
-                </div>
-            );
+        if (!cloudFrontUrl) {
```

Comment:
Return null always jumps out to me, could this be replaced with:
`if (cloudFrontUrl) {
  return <PdfViewer fileUrl={cloudFrontUrl} />;
}`

Reply:
Discussed this together and came up with a ternary.